pytorch3d/io/obj_io.py

```diff
@@ -728,6 +741,7 @@ def save_obj(
     if path_manager is None:
         path_manager = PathManager()
 
+    save_normals = all([n is not None for n in [verts_normals, faces_normals]])
```

Comment:
Is the idea that, if they want, the user can supply verts_normals or faces_normals or both? I think so, and I think the logic for dealing with verts_normals and the logic for dealing with faces_normals should have nothing to do with each other. There's no need for a `save_normals` flag which makes them interact.

Reply:
Hi, the logic is that both `verts_normal` and `faces_normal` are required to save normals in the OBJ file. `faces_normal` are used to index the `verts_normal`, and for the case when each vertex has only one corresponding normal, it should be the same with `faces`.

Reply:
Sure, I've changed it. I took the name similar to `verts_uvs` and `faces_uvs`, but it is indeed a bit misleading.